landscape.yml

```diff
@@ -912,6 +917,11 @@ landscape:
             homepage_url: https://github.com/tencent
             logo: tencent.svg
             crunchbase: https://www.crunchbase.com/organization/tencent
+          - item:
+            name: TomTom (Adopter)
+            homepage_url: https://www.tomtom.com
+            logo: tomtom.svg
+            crunchbase: https://www.crunchbase.com/organization/tomtom-international
```

Comment:
Does TomTom have an OSPO or open source public-facing site (or GitHub/GitLab page)we could use instead of  https://www.tomtom.com? If not, it's fine 👍

Reply:
Hi Ana, yes ;-) It is in development but live. I've added it to this PR.